release-notes/opensearch-sql.release-notes-1.3.0.0.md

```diff
@@ -0,0 +1,41 @@
+### Version 1.2.4.0 Release Notes
+
+Compatible with OpenSearch and OpenSearch Dashboards Version 1.3.0
+
+### Enhancements
+* PPL integration with AD and ml-commons ([#468](https://github.com/opensearch-project/sql/pull/468))
```

Comment:
also not sure if this is going in? this and 411 can be features

Reply:
@penghuo Is this not going in?

Reply:
yes, parse command release in 1.3

Reply:
#468, not parse, is AD ml-commons going in?